test/xcommon_cmake/src/main.c

```diff
@@ -0,0 +1,32 @@
+
+#include "xmath/xmath.h"
+
+int main(void)
+{
+  // Just reference a few symbols to make sure that they can be linked in
+  headroom_t (*ptr0) (int32_t *, const int32_t*, const int32_t*, const unsigned, const right_shift_t, const right_shift_t);
+  ptr0 = &vect_s32_add;
+  printf("prt0 = %lx\n", (long)ptr0);
```

Comment:
Ignore this if it's too pedantic :)
Throughout this file you have function pointers calls `ptrX` and the prints say `prtX`.